notebooks/03-score_model_demo.ipynb

```diff
@@ -0,0 +1,458 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "537c04d1",
+   "metadata": {},
+   "source": [
+    "# Scoring a model and logging the results to Weight and Biases\n",
+    "\n",
+    "This notebook demonstrates how to score your model on the test_2022 dataset and upload the results to Weights and Biases (wandb).\n",
+    "\n",
+    "Before being able to upload your results to wandb, you will need to be added to the wandb team set up for this project. Please contact the research engineering team for this.\n",
+    "\n",
+    "You will also need to have downloaded or have access to the test_2022 dataset - it is available in Baskerville in the shared data folder. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "957ad082",
+   "metadata": {},
+   "source": [
+    "### Step 1. Create a class to run your model\n",
+    "\n",
+    "In this notebook we'll make a blurry persistence baseline model. This model will take the most recent satellite image, apply some gaussian blur, and then predict this same blurry image for the future time steps. \n",
+    "\n",
+    "This is a somewhat contrived example, a better baseline likely wouldn't blur the image before projecting it into the future. However, this version will allow us to show the working of the library better. The normal version of persistence is a little too simple to show things effectively"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "1599f122",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# First we'll load a sample from the dataset. We'll use this sample later\n",
+    "from cloudcasting.dataset import SatelliteDataset\n",
+    "\n",
+    "dataset = SatelliteDataset(\n",
+    "    zarr_path=\"path/to/satellite/data/2020_nonhrv.zarr\",\n",
+    "    start_time=\"2020-06-01\",\n",
+    "    end_time=\"2020-06-15\",\n",
+    "    history_mins=0,\n",
+    "    forecast_mins=180,\n",
+    "    sample_freq_mins=15,\n",
+    ")\n",
+    "\n",
+    "# Grab a single sample\n",
+    "X, y = dataset[1000]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e5dbb2f",
+   "metadata": {},
+   "source": [
+    "#### - Quick detour: Lets show what the blur part of this persistence model actually does\n",
+    "\n",
+    "This isn't particularly important, so feel free to skip this\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9e330f92",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
```

Comment:
I suggest clearing image/video outputs from notebooks before checking them in unless you're certain that they're never going to change.

N.B. Changing this would involve either making this PR a "squash" merge or editing the history to change the commit that added this.

Reply:
I am fairly certain that the images won't change in the future. I know it could make the repo really heavy by including them, but I think they're useful as a tutorial for the users. 

Maybe we could include them for now, but if we'll reconsider in the future if we ever find we do need to replace them?

Reply:
Yeah - the main problem with images is that anyone rerunning the notebook is likely to make an image that's not bitwise-identical even if it has the same content. If we're confident that people aren't going to rerun and commit the notebook from their new run then it's fine.